cmd/driver/install/install.go

```diff
@@ -64,16 +65,12 @@ func NewDriverInstallCmd(ctx context.Context, opt *options.Common) *cobra.Comman
 		Long: `[Preview] Install previously configured driver, either downloading it or attempting a build.
 ** This command is in preview and under development. **`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			driver, err := config.Driverer()
-			if err != nil {
-				return err
-			}
 			// If empty, try to load it automatically from /usr/src sub folders,
 			// using the most recent (ie: the one with greatest semver) driver version.
-			if driver.Version == "" {
-				driver.Version = loadDriverVersion()
+			if o.Driver.Version == "" {
```

Comment:
I don't know if we should move this thing in `driver` command too; perhaps it makes sense after all.

Reply:
This way, we always try hard to discover the driver version when empty.

Reply:
Done in latest commit.